### DISCUSSION TITLE
Add a "Set Home Dir" Event action supporting Placeholders

### DISCUSSION THREAD

**Discussion by Cheekie25** [drakkan/sftpgo#1447] [Ideas]
I think it would be interesting to add an Event Action that would allow us to set the Home Dir of the user using Placeholders, especillay custom ones.
The main use case I see is when login with an Identity Provider.
This would allow us to use a Custom Placeholder such as fullname to set the Home Directory instead of the username.

**Comment by drakkan** [comment]
Hello, I think this is already supported. I don't have the time/motivation to test it myself or provide detailed instructions, sorry
